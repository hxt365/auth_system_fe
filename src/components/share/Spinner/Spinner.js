import React from 'react';
import { Spin } from 'antd';
import './Spinner.scss';

function Spinner() {
  return (
    <div className="spinner">
      <Spin size="large" />
    </div>
  );
}

export default Spinner;
