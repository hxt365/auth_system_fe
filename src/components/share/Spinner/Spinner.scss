.spinner {
  text-align: center;
  background: rgba(0, 0, 0, 0.05);
  border-radius: 4px;
  margin-bottom: 2rem;
  padding: 3rem 5rem;
  margin: 2rem 0;
}
