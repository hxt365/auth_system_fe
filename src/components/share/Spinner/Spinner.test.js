import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Spinner from './Spinner';

describe('test Spinner', () => {
  it('test snapshot', () => {
    const wrapper = shallow(<Spinner />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
