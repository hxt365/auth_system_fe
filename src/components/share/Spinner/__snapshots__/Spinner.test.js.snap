// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Spinner test snapshot 1`] = `
<div
  className="spinner"
>
  <Spin
    size="large"
    spinning={true}
    wrapperClassName=""
  />
</div>
`;
